terraform.yml

```diff
@@ -66,10 +66,11 @@ parameters:
     default: "microsoft-avere:vfxt:avere-vfxt-node:latest"
     values:
       - "microsoft-avere:vfxt:avere-vfxt-node:latest"
+      - "microsoft-avere:vfxt:avere-vfxt-node:8.0.1"
+      - "microsoft-avere:vfxt:avere-vfxt-node:6.3.0"
+      - "microsoft-avere:vfxt:avere-vfxt-node:6.2.0"
       - "microsoft-avere:vfxt:avere-vfxt-node:6.0.0"
       - "microsoft-avere:vfxt:avere-vfxt-node:5.3.61"
```

Comment:
Similar to a previous comment, I'd recommend removing 6.2.0, 6.0.0, and 5.* from here.

Reply:
got it!